src/main/java/com/github/skjolber/jackson/jsh/indenter/SyntaxHighlighterNoopIndenter.java

```diff
@@ -0,0 +1,49 @@
+package com.github.skjolber.jackson.jsh.indenter;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.github.skjolber.jackson.jsh.highlighter.SyntaxHighlighter;
+
+import java.io.IOException;
+
+/**
+ * An indenter that prints json to a single line (un-prettified) and maintains the ANSI text colors
+ */
+public class SyntaxHighlighterNoopIndenter extends DefaultPrettyPrinter.NopIndenter implements SyntaxHighlighterStyleIndenter {
```

Comment:
Perhaps this whole class would be avoided by passing a boolean flag to SyntaxHighlighterObjectIndenter?